[2022-06-30 02:12:20.091] [remoteagent] [info] Extension host agent started.
[2022-06-30 02:12:20.590] [remoteagent] [info] [127.0.0.1][9cac0b0f][ManagementConnection] New connection established.
[2022-06-30 02:12:20.592] [remoteagent] [info] [127.0.0.1][8841f935][ExtensionHostConnection] New connection established.
[2022-06-30 02:12:20.609] [remoteagent] [info] [127.0.0.1][8841f935][ExtensionHostConnection] <148472> Launched Extension Host Process.
[2022-06-30 02:27:06.234] [remoteagent] [info] [127.0.0.1][8841f935][ExtensionHostConnection] The client has reconnected.
[2022-06-30 02:27:10.800] [remoteagent] [info] [127.0.0.1][9cac0b0f][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-30 02:27:10.804] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-30 05:03:06.628] [remoteagent] [info] [127.0.0.1][9cac0b0f][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-30 05:03:06.629] [remoteagent] [info] [127.0.0.1][14591159][ManagementConnection] New connection established.
[2022-06-30 05:03:06.636] [remoteagent] [info] [127.0.0.1][4cb9340b][ExtensionHostConnection] New connection established.
[2022-06-30 05:03:06.650] [remoteagent] [info] [127.0.0.1][4cb9340b][ExtensionHostConnection] <149764> Launched Extension Host Process.
[2022-06-30 05:03:17.676] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-30 05:03:17.678] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-30 05:08:06.792] [remoteagent] [info] [127.0.0.1][9cac0b0f][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-06-30 05:08:06.872] [remoteagent] [info] [127.0.0.1][8841f935][ExtensionHostConnection] <148472> Extension Host Process exited with code: 0, signal: null.
[2022-06-30 05:28:33.714] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/fred)
[2022-06-30 05:28:33.715] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[2022-06-30 08:08:09.010] [remoteagent] [info] [127.0.0.1][14591159][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-30 08:08:09.014] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-30 08:10:40.293] [remoteagent] [info] [127.0.0.1][14591159][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-30 08:10:40.295] [remoteagent] [info] [127.0.0.1][2ce6826d][ManagementConnection] New connection established.
[2022-06-30 08:10:40.450] [remoteagent] [info] [127.0.0.1][9a577943][ExtensionHostConnection] New connection established.
[2022-06-30 08:10:40.468] [remoteagent] [info] [127.0.0.1][9a577943][ExtensionHostConnection] <196023> Launched Extension Host Process.
[2022-06-30 08:10:46.262] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-30 08:10:46.262] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-30 08:15:40.435] [remoteagent] [info] [127.0.0.1][14591159][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-06-30 08:15:40.478] [remoteagent] [info] [127.0.0.1][4cb9340b][ExtensionHostConnection] <149764> Extension Host Process exited with code: 0, signal: null.
